helm/values.yaml

```diff
@@ -0,0 +1,210 @@
+# System
+replicaCount: 1
+strategyType: Recreate
+
+nameOverride: ""
+fullnameOverride: ""
+
+images:
+  ucentralsec:
+    repository: tip-tip-wlan-cloud-ucentral.jfrog.io/ucentralsec
+    tag: main
+    pullPolicy: Always
+#    regcred:
+#      registry: tip-tip-wlan-cloud-ucentral.jfrog.io
+#      username: username
+#      password: password
+
+services:
+  ucentralsec:
+    type: LoadBalancer
+    ports:
+      restapi:
+        servicePort: 16001
+        targetPort: 16001
+        protocol: TCP
+      restapiinternal:
+        servicePort: 17001
+        targetPort: 17001
+        protocol: TCP
+
+checks:
+  ucentralsec:
+    liveness:
+      httpGet:
+        path: /
+        port: 16101
+    readiness:
+      httpGet:
+        path: /
+        port: 16101
+
+ingresses:
+  restapi:
+    enabled: false
+    annotations: {}
+      # kubernetes.io/ingress.class: nginx
+      # kubernetes.io/tls-acme: "true"
+    hosts:
+    - restapi.chart-example.local
+    paths:
+    - path: /
+      serviceName: ucentralsec
+      servicePort: restapi
+
+volumes:
+  ucentralsec:
+    - name: config
+      mountPath: /ucentral-data/ucentralsec.properties
+      subPath: ucentralsec.properties
+      # Template below will be rendered in template
+      volumeDefinition: |
+        secret:
+          secretName: {{ include "ucentralsec.fullname" . }}-config
+    - name: certs
+      mountPath: /ucentral-data/certs
+      volumeDefinition: |
+        secret:
+          secretName: {{ include "ucentralsec.fullname" . }}-certs
+    # Change this if you want to use another volume type
+    - name: persist
+      mountPath: /ucentral-data/persist
+      volumeDefinition: |
+        persistentVolumeClaim:
+          claimName: {{ template "ucentralsec.fullname" . }}-pvc
+
+resources: {}
+  # We usually recommend not to specify default resources and to leave this as a conscious
+  # choice for the user. This also increases chances charts run on environments with little
+  # resources, such as Minikube. If you do want to specify resources, uncomment the following
+  # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
+  # requests:
+  #  cpu: 100m
+  #  memory: 128Mi
+  # limits:
+  #  cpu: 100m
+  #  memory: 128Mi
+
+nodeSelector: {}
+
+tolerations: []
+
+affinity: {}
+
+persistence:
+  enabled: true
+  # storageClassName: "-"
+  accessModes:
+    - ReadWriteOnce
+  size: 10Gi
+  annotations: {}
+
+# Application
+public_env_variables:
+  UCENTRALSEC_ROOT: /ucentral-data
```

Comment:
Should we change this to `/ucentralsec-data`? At least this directory is created in the Dockerfile.

Reply:
Done